src/BigFish/PaymentGateway/Config.php

```diff
@@ -21,6 +21,7 @@
  * @property string $moduleName Module name
  * @property string $moduleVersion Module version
  * @property string $gatewayUrlTest Test service URL
+ * @property string $gatewayProxy Test service URL
```

Comment:
Please, rename the "Test service URL" to "Proxy address used for the api requests"